vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -921,6 +925,26 @@ components:
         id:
           type: string
 
+    VdpaConfig:
+      required:
+      - path
+      - num_queues
+      type: object
+      properties:
+        path:
+          type: string
+        num_queues:
+          type: integer
+          default: 1
+        iommu:
```

Comment:
I think you didn't implement the iommu support yet so would be good to leave this out for now?